integration/sql/impl/Cargo.toml

```diff
@@ -14,4 +14,4 @@ crate-type = ["rlib"]
 [dependencies]
 anyhow = {workspace = true}
 
-sqlparser = {git = "https://github.com/OpenLineage/sqlparser-rs/", branch = "release"}
\ No newline at end of file
+sqlparser = {git = "https://github.com/OpenLineage/sqlparser-rs/", branch = "fix_snowflake_stage_no_parens"}
```

Comment:
Why do we want to rely on feature branch? If there is any particular reason to do that, pls specify PR that we need to merge in another repo. Otherwise it may be difficult to change this in future.

Reply:
Because if we merge the sqlparser-rs changes first here: https://github.com/OpenLineage/sqlparser-rs/pull/5 it would break the main: so the algorithm is

- do the sqlparser change on a branch
- make changes in the OpenLineage repo, pointing to the branch
- merge sqlparser changes to our fork
- move the changes back to the release branch on OpenLineage repo